doc/deprecated.rst

```diff
@@ -39,6 +39,10 @@ Tags
 * The ``filter`` tag is deprecated in Twig 2.9. Use the ``apply`` tag instead
   (the ``Twig\TokenParser\FilterTokenParser`` classes is also deprecated).
 
+* Adding an ``if`` condition on a ``for`` tag is deprecated in Twig 2.10. Use a
+  ``filter`` filter or a condition inside the ``if`` instead (if your condition
```

Comment:
```suggestion
  ``filter`` filter or a condition inside the ``loop`` instead (if your condition
```

Not sure if typo?

Reply:
indeed, I've reworded the sentence to read: "or an `if` condition inside the ``for`` body instead"

Reply:
Great! 👍 

(PS: Oh yeah, `for` oops 😅 )